diff --git a/chrome/browser/chrome_browser_main.cc b/chrome/browser/chrome_browser_main.cc
index cbf3b6c563..a81866c7f0 100644
--- a/chrome/browser/chrome_browser_main.cc
+++ b/chrome/browser/chrome_browser_main.cc
@@ -225,6 +225,7 @@
 #include "chrome/browser/infobars/infobar_features.h"
 #include "chrome/browser/mac/chrome_browser_main_extra_parts_mac.h"
 #include "chrome/browser/shutdown_watchdog_mac.h"
+#include "fiber/browser/fiber_browser_main_extra_parts.h"
 #include "chrome/browser/ui/browser_window/public/browser_window_interface.h"
 #include "chrome/browser/ui/browser_window/public/global_browser_collection.h"
 #include "chrome/browser/ui/cocoa/keystone_infobar_delegate.h"
@@ -821,6 +822,7 @@ std::unique_ptr<content::BrowserMainParts> ChromeBrowserMainParts::Create(
 
 #if BUILDFLAG(IS_MAC)
   main_parts->AddParts(std::make_unique<ChromeBrowserMainExtraPartsMac>());
+  main_parts->AddParts(std::make_unique<fiber::FiberBrowserMainExtraParts>());
 #endif
 
 #if BUILDFLAG(IS_CHROMEOS)
