diff --git a/chrome/browser/browser_resources.grd b/chrome/browser/browser_resources.grd
index 6309487fa3..632b797722 100644
--- a/chrome/browser/browser_resources.grd
+++ b/chrome/browser/browser_resources.grd
@@ -90,15 +90,8 @@
       </if>
     </structures>
     <includes>
-      <if expr="is_win or is_macosx or is_linux">
-        <!-- First Run audio resources -->
-        <include name="IDR_INTRO_SOUND_ALL_SET_FLAC" file="resources\intro\sounds\all_set.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_AMBIENT_FLAC" file="resources\intro\sounds\ambient.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_FEATURE_SHOWCASE_AMBIENT_FLAC" file="resources\intro\sounds\feature_showcase_ambient.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_FEATURE_SHOWCASE_PROGRESS_FLAC" file="resources\intro\sounds\feature_showcase_progress.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_LOGO_FLAC" file="resources\intro\sounds\logo.flac" type="BINDATA" />
-        <include name="IDR_INTRO_SOUND_WELCOME_BACK_FLAC" file="resources\intro\sounds\welcome_back.flac" type="BINDATA" />
-      </if>
+      <!-- Fiber: not First Run's audio, since Fiber never shows Chrome's
+           First Run. -->
       <if expr="enable_src_internal">
         <include name="IDR_APC_PROMPTS_JSON" file="../../internal/password_manager/apc_prompts.json" type="BINDATA" />
         <include name="IDR_SEARCH_ENGINE_HISTORICAL_SEARCH_URLS_JSON" file="../../third_party/search_engines_data/resources_internal/definitions/historical_search_urls.json" type="BINDATA" />
