diff --git a/chrome/browser/ui/webui/help/version_updater_basic.cc b/chrome/browser/ui/webui/help/version_updater_basic.cc
index ebcef5637b..b7eed9d41b 100644
--- a/chrome/browser/ui/webui/help/version_updater_basic.cc
+++ b/chrome/browser/ui/webui/help/version_updater_basic.cc
@@ -5,6 +5,7 @@
 #include <memory>
 #include <string>
 
+#include "build/build_config.h"
 #include "chrome/browser/ui/webui/help/version_updater.h"
 #include "chrome/browser/upgrade_detector/upgrade_detector.h"
 
@@ -25,6 +26,10 @@ class VersionUpdaterBasic : public VersionUpdater {
                               : DISABLED;
     callback.Run(status, 0, false, false, std::string(), 0, std::u16string());
   }
+#if BUILDFLAG(IS_MAC)
+  // Fiber: there's no updater to promote.
+  void PromoteUpdater() override {}
+#endif
 };
 
 }  // namespace
