diff --git a/chrome/browser/devtools/devtools_window_views.cc b/chrome/browser/devtools/devtools_window_views.cc
index 9331bad3ad..e6b7738ed7 100644
--- a/chrome/browser/devtools/devtools_window_views.cc
+++ b/chrome/browser/devtools/devtools_window_views.cc
@@ -14,6 +14,7 @@
 #include "components/web_modal/web_contents_modal_dialog_manager.h"
 #include "content/public/browser/keyboard_event_processing_result.h"
 #include "content/public/browser/render_process_host.h"
+#include "fiber/browser/hooks/devtools_dock.h"
 #include "printing/buildflags/buildflags.h"
 
 #if BUILDFLAG(ENABLE_PRINT_PREVIEW)
@@ -139,6 +140,8 @@ void DevToolsWindow::UpdateBrowserWindow() {
           DevtoolsUIController::From(inspected_browser)) {
     devtools_ui_controller->UpdateDevtools(inspected_web_contents);
   }
+  // Fiber: Fiber's windows have no DevtoolsUIController.
+  fiber::UpdateDevTools(inspected_browser, inspected_web_contents);
 }
 
 void DevToolsWindow::RegisterModalDialogManager(
