diff --git a/chrome/browser/app_controller_mac.h b/chrome/browser/app_controller_mac.h
index 62f68487c9..8c12bf50b2 100644
--- a/chrome/browser/app_controller_mac.h
+++ b/chrome/browser/app_controller_mac.h
@@ -23,7 +23,6 @@
 #include "components/prefs/pref_change_registrar.h"
 
 class BookmarkMenuBridge;
-@class ConfirmQuitPanelController;
 class GURL;
 class HistoryMenuBridge;
 class Profile;
@@ -156,7 +155,6 @@ typedef NS_ENUM(NSInteger, ConfirmQuitResult) {
 - (BOOL)windowHasBrowserTabs:(NSWindow*)window;
 
 // Testing API.
-- (ConfirmQuitPanelController*)confirmQuitPanelControllerForTesting;
 - (void)setCmdWMenuItemForTesting:(NSMenuItem*)menuItem;
 - (void)setShiftCmdWMenuItemForTesting:(NSMenuItem*)menuItem;
 
