diff --git a/third_party/ffmpeg/chromium/config/Chromium/mac/x64/config_components.h b/third_party/ffmpeg/chromium/config/Chromium/mac/x64/config_components.h
index 6d7d59f..68884b8 100644
--- a/third_party/ffmpeg/chromium/config/Chromium/mac/x64/config_components.h
+++ b/third_party/ffmpeg/chromium/config/Chromium/mac/x64/config_components.h
@@ -1016,7 +1016,7 @@
 #define CONFIG_WMV3_NVDEC_CUARRAY_HWACCEL 0
 #define CONFIG_WMV3_VAAPI_HWACCEL 0
 #define CONFIG_WMV3_VDPAU_HWACCEL 0
-#define CONFIG_AAC_PARSER 0
+#define CONFIG_AAC_PARSER 1
 #define CONFIG_AAC_LATM_PARSER 0
 #define CONFIG_AC3_PARSER 0
 #define CONFIG_ADX_PARSER 0
@@ -1709,7 +1709,7 @@
 #define CONFIG_AMOVIE_FILTER 0
 #define CONFIG_MOVIE_FILTER 0
 #define CONFIG_AA_DEMUXER 0
-#define CONFIG_AAC_DEMUXER 0
+#define CONFIG_AAC_DEMUXER 1
 #define CONFIG_AAX_DEMUXER 0
 #define CONFIG_AC3_DEMUXER 0
 #define CONFIG_AC4_DEMUXER 0
