diff --git a/third_party/devtools-frontend/src/front_end/entrypoints/main/MainImpl.ts b/third_party/devtools-frontend/src/front_end/entrypoints/main/MainImpl.ts
index 657383f8..efbe57fc 100644
--- a/third_party/devtools-frontend/src/front_end/entrypoints/main/MainImpl.ts
+++ b/third_party/devtools-frontend/src/front_end/entrypoints/main/MainImpl.ts
@@ -798,26 +798,6 @@ export class MainMenuItem implements UI.Toolbar.Provider {
           ${i18nString(UIStrings.dockSide)}
         </span>
         <devtools-toolbar @mousedown=${(event: Event) => event.consume()}>
-          <devtools-button class="toolbar-button"
-                           jslog=${VisualLogging.toggle().track({click: true}).context('current-dock-state-undock')}
-                           title=${i18nString(UIStrings.undockIntoSeparateWindow)}
-                           aria-label=${i18nString(UIStrings.undockIntoSeparateWindow)}
-                           .iconName=${'dock-window'}
-                           .toggled=${dockController.dockSide() === UI.DockController.DockState.UNDOCKED}
-                           .toggledIconName=${'dock-window'}
-                           .toggleType=${Buttons.Button.ToggleType.PRIMARY}
-                           .variant=${Buttons.Button.Variant.ICON_TOGGLE}
-                           @click=${setDockSide.bind(null, UI.DockController.DockState.UNDOCKED)}></devtools-button>
-          <devtools-button class="toolbar-button"
-                           jslog=${VisualLogging.toggle().track({click: true}).context('current-dock-state-left')}
-                           title=${i18nString(UIStrings.dockToLeft)}
-                           aria-label=${i18nString(UIStrings.dockToLeft)}
-                           .iconName=${'dock-left'}
-                           .toggled=${dockController.dockSide() === UI.DockController.DockState.LEFT}
-                           .toggledIconName=${'dock-left'}
-                           .toggleType=${Buttons.Button.ToggleType.PRIMARY}
-                           .variant=${Buttons.Button.Variant.ICON_TOGGLE}
-                           @click=${setDockSide.bind(null, UI.DockController.DockState.LEFT)}></devtools-button>
           <devtools-button class="toolbar-button"
                            jslog=${VisualLogging.toggle().track({click: true}).context('current-dock-state-bottom')}
                            title=${i18nString(UIStrings.dockToBottom)}
